<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUser</class>
 <widget class="QDialog" name="AddUser">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>375</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>210</width>
    <height>375</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>210</width>
    <height>375</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AddUser</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>183</width>
     <height>358</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nazwa użytkownika</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="adduserLogin">
      <property name="minimumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Imię</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="adduserName">
      <property name="minimumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Nazwisko</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="adduserSurname">
      <property name="minimumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Hasło</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="adduserPassword">
      <property name="minimumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Potwierdź hasło</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="adduserConfirmpassword">
      <property name="minimumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="minimumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>181</width>
        <height>31</height>
       </size>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>adduserLogin</tabstop>
  <tabstop>adduserName</tabstop>
  <tabstop>adduserSurname</tabstop>
  <tabstop>adduserPassword</tabstop>
  <tabstop>adduserConfirmpassword</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
